src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -39,7 +40,7 @@ public static class Win32ApplicationExtensions
     public class Win32PlatformOptions
     {
         public bool UseDeferredRendering { get; set; } = true;
-        public bool AllowEglInitialization { get; set; } = true;
+        public bool AllowEglInitialization { get; set; } = Win32Platform.WindowsVersion > new Version(6, 1);
```

Comment:
Should it be some special property that tells "WinRT backend supported"? Otherwise it can be easy to forget update min.version in both places.

Reply:
ANGLE seems way too unstable on Win7 to be enabled by default.

Reply:
This is odd. Qt was using ANGLE as the backend for its QML UI on Windows 7 forever.

Reply:
And we've been getting error reports for Win7 only.

Reply:
It's most likely on our end, but we don't known what we are doing wrong and it's not worth to investigate a out-of-life platform.

Reply:
That I can totally agree with! :-D